draft-ietf-6man-slaac-renum-08.xml

```diff
@@ -360,18 +414,20 @@ This is to align the Router Lifetime with the recommendations in <xref target="R
 </t>
 </list>
 </t>
-
-
-
-
-
+<t>
+While allowing updates to the valid lifetime in RAs could enable an attacker to invalidate addresses by setting the valid lifetime to zero, this doesn't significantly worsen the security situation. An attacker capable of sending rogue RAs already has the power to disrupt connectivity by manipulating other parameters, such as gateway or DNS information.  Therefore, accepting a zero lifetime doesn't make the system more vulnerable than it already is in the presence of such an attacker.
+</t>
+<t>
+In scenarios where RA-based attacks are of concern, proper mitigations such as RA-Guard <xref target="RFC6105"/> <xref target="RFC7113"/> or SEND <xref target="RFC3971"/> should be implemented. 
+</t>
```

Comment:
I'd probaby s/proper//

-- the thing here is that SEND is probably not much deplyable. And, if you care about DoS, RA-Guard will hopefully mitigate the forged RA packets. But then an attacker can still forge e.g. NAs ane.g. map the router's link-local IPv6 address to a non-ecisting link-layer address, for DoS purposes.  (stricly speaking RA-Guard will mitigate "RA-based attacks", as noted, though, but not ND attacks, which may also employ other ND messages...)

Reply:
> I'd probaby s/proper//

Done.

As this paragraph is about a specific RA-based attack, I think it's OK not to focus on ND-based.
We may mention p2p isolation as another approach, but I think it is not exactly in scope for this paragraph...

Reply:
Focusing on RA-based attacks is fair enough, as you note.  (the threat model for ND is that you trust onlink nodes and ND packets, anyway).